<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceDialog</class>
 <widget class="QDialog" name="DeviceDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Device</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" colspan="4">
      <widget class="QListWidget" name="list_widget">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_add_all_devices">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Only active when Connected</string>
       </property>
       <property name="text">
        <string>Add All Connected Devices</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_add_device">
       <property name="text">
        <string>Add Device</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
